plugins/git/gitlab_statistics

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+# -*- python -*-
+# -*- coding: utf-8 -*-
+#
+# Plugin to monitor Gitlab status
+#
+# Usage: Place in /etc/munin/plugins/ (or link it there using ln -s)
+#   Add this to your /etc/munin/plugin-conf.d/munin-node:
+#       [gitlab]
+#       user nobody
+#       logarithmic 1
+#       hostname gitlab.example.com # required
+#       token YourPrivateTokenHere # required
+#
+# 2019-10-02: v 1.0 pcy <pcy.ulyssis.org>: created
+#
+# Parameters understood:
+#
+#   config   (required)
+#   autoconf (optional - used by munin-config)
+#
+# magic markers:
+#%# family=auto
+#%# capabilities=autoconf
+
+import os
+import requests
+import sys
+
+
+def weakbool(x):
+    u = x.upper().strip()
+    if u == "FALSE" or u == "NO" or u == "N":
+        return False
+    try:
+        return bool(int(x))
+    except ValueError:
+        return bool(x)
+
+
+url = None
+if 'hostname' in os.environ and 'token' in os.environ:
+    url = "https://" + os.getenv('hostname') \
+        + "/api/v4/application/statistics?private_token=" \
+        + os.getenv('token')
+
+logarithmic = weakbool(os.getenv('logarithmic', 'N'))
+
+
+def reqjson():
+    connection = requests.get(url)
+    outputjson = connection.json()
+    status = connection.status_code
+
+    if status != 200:
+        print("???")
+        sys.exit(1)
+
+    return outputjson
+
+
+def autoconf():
+    print("no" if url is None else "yes")
+
+
+def config():
+    print("""\
+graph_title GitLab statistics
+graph_vlabel amount
+""" + ("graph_args --logarithmic" if logarithmic else "") + """
+graph_category gitlab\
```

Comment:
It is recommend to use one of the [well-known categories](http://guide.munin-monitoring.org/en/latest/reference/graph-category.html#well-known-categories) - here `devel` would be suitable.

Reply:
Changed that.